### PR TITLE
LPS-205374 Unique field constraint is not site-scoped

### PR DESCRIPTION
**Pull request by pedro-oliveira446**
[LPS-205374](https://liferay.atlassian.net/browse/LPS-205374)

This PR is necessary because of this backport: https://github.com/brianchandotcom/liferay-portal/pull/146332 .

### CODE REVIEW

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:upgrade

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v9_0_1/ObjectFolderUpgradeProcess.java [hunk 1]
```diff
@@ -54,6 +54,18 @@ protected void doUpgrade() throws Exception {
...
+									company.getCompanyId(),
+									ObjectFolder.class.getName(),
+									ResourceConstants.SCOPE_INDIVIDUAL,
+									String.valueOf(
+										resultSet.getLong("objectFolderId")),
+									guestRole.getRoleId(), ActionKeys.VIEW)) {
+
+							continue;
+						}
+
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v9_0_1/ObjectFolderUpgradeProcess.java:68]: @pedro-oliveira446 Is this really necessary? I believe setResourcePermissions can be executed more than once.

cc @MarinhoFeliphe

**Review by gabrielwas — CHANGES_REQUESTED** [review]
@pedro-oliveira446 check my single comment. The rest is fine to me. Thx.

**Comment by pedro-oliveira446** [comment]
ci:test:sf

**Comment by pedro-oliveira446** [comment]
ci:test:relevant

**Comment by pedro-oliveira446** [comment]
ci:test:object

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v9_0_1/ObjectFolderUpgradeProcess.java (hunk 1 above)
Comment (pedro-oliveira446) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v9_0_1/ObjectFolderUpgradeProcess.java]: @gabrielwas I discussed this with @MarinhoFeliphe  and we agree with you. Thanks.

**Comment by pedro-oliveira446** [comment]
Resente [here](https://github.com/brianchandotcom/liferay-portal/pull/146426)

**Pull request closed without merging**